DSCResources/MSFT_xPendingReboot/MSFT_xPendingReboot.psm1

```diff
@@ -57,33 +57,58 @@ Function Test-TargetResource
      param
     (
     [Parameter(Mandatory=$true)]
-    [string]$Name
+    [string]$Name,
+    [bool]$SkipComponentBasedServicing,
```

Comment:
The defaults for a bool is $false, so this will be a significant behavior change.  Suggestion, default to true.